### PR TITLE
Updates icons to tooltips

### PR DESCRIPTION
**Pull request by bruugey**
Also fixes component display names

### CODE REVIEW

File: layouts/ComponentLayout.tsx [hunk 1]
```diff
@@ -89,6 +90,74 @@ const tabStyles = css`
...
+  if (!href) {
+    return null;
+  }
+
+  return (
+    <Tooltip
+      trigger={
+        <IconButton
+          key={type}
+          aria-label="View in CodeSandbox"
```
Comment (shaneeza) [inline layouts/ComponentLayout.tsx:145]: Needs to be updated

File: layouts/ComponentLayout.tsx [hunk 2]
```diff
@@ -89,6 +90,74 @@ const tabStyles = css`
...
+          as="a"
+          target="_blank"
+          rel="noopener noreferrer"
+          style={{ marginRight: '8px' }}
```
Comment (shaneeza) [inline layouts/ComponentLayout.tsx:149]: Do we want margin-right on all of these?

Comment (bruugey) [reply layouts/ComponentLayout.tsx:149]: that's what we had before

File: layouts/ComponentLayout.tsx (hunk 1 above)
Comment (bruugey) [reply layouts/ComponentLayout.tsx]: fixed

File: layouts/ComponentLayout.tsx (hunk 2 above)
Comment (spark33) [reply layouts/ComponentLayout.tsx:149]: It's a nit, but it would be better to use `gap: ${spacing[1]}px` now that there are 3 elements

**Review by spark33 — APPROVED** [review]

**Pull request merged**